### DISCUSSION TITLE
Fertilizer Additive Selection

### DISCUSSION THREAD

**Discussion by RonColes** [holos-aafc/Holos#205] [General]
I am interested to know how the selection of a fertilizer additive affects algorithms or emission calculations? I looked through the algorithm document and could not find a reference. Thank you.

**Comment by RonColes** [comment]
Hi Aaron,

Couple of questions:


  1.  How does Holos use the “Season of Application” when adding a fertilizer?
  2.  Is there any points in Holos where a fertilizer source (country) does not matter?

Thanks


***@***.***
Ron Coles
Senior Project Specialist

Mobile 902-393-0841
Web www.lpconsulting.ca<http://www.lpconsulting.ca/>
PO Box 193
Charlottetown, PE  C1A 7K4

**Comment by RonColes** [comment]
So Holos assumes all fertilizers are made in the same country or have the same production value?


***@***.***
Ron Coles
Senior Project Specialist

Mobile 902-393-0841
Web www.lpconsulting.ca<http://www.lpconsulting.ca/>
PO Box 193
Charlottetown, PE  C1A 7K4

**Reply by RKroebel** [reply]
hi Ron, we use NorthAmerican numbers right now, as reported in the Bentrup paper, there is also numbers for other continents, but it seemed unlikely that farmers would know what continent their fertilizer is coming from, and we'd have to ask that. So for ease of use, we decided to assume all fertilizer is Canada/US made. If we were to consider sources from Russia (for instance), we'd also have to account for transport. This is something that can be done in the future, but not for the current version (as we are attempting to close it up).
